lib/mpl_toolkits/mplot3d/axes3d.py

```diff
@@ -1343,29 +1318,6 @@ def grid(self, visible=True, **kwargs):
         self._draw_grid = visible
         self.stale = True
 
-    def locator_params(self, axis='both', tight=None, **kwargs):
```

Comment:
Perhaps I've missed some context, but why is this and some other public API being removed?

Reply:
They are now directly inherited from the base `_AxesBase` class, which iterates over all axises and can thus handle both 2D and 3D.